### PR TITLE
feature/update sending device descriptions notes

### PR DESCRIPTION
**Pull request by saschadoemer**

### CODE REVIEW

File: docs/commands/endpoint.adoc [hunk 1]
```diff
@@ -230,11 +230,16 @@ Errors will be reported using `ACK_WITH_FAILURE`. For a list of possible errors,
...
+== Sending a teamset using device descriptions
 
 === Definition
 
-The teamset describes a list of devices attached to the communication unit. The most common devices are agricultural machines connected to the CU as ISO11783-10 TaskControllers. For more information on ISO11783-10, please refer to https://aef-online.org
+The teamset describes a list of devices attached to the communication unit. The most common devices are agricultural machines connected to the CU as ISO11783-10 task controllers. For more information on ISO11783-10, please refer to https://aef-online.org
+
+[IMPORTANT]
+====
+Please note that the device description has to be sent every time before the maschine starts sending live telemetry data to ensure, that each other endpoint is able to update their formerly received devices descriptions and new endpoints get the latest updates as well.
```
Comment (Frank-Wiebeler) [inline docs/commands/endpoint.adoc:241]: machine

Comment (Frank-Wiebeler) [reply docs/commands/endpoint.adoc:241]: Please note that the device description has to be sent every time before the machine starts sending live telemetry data to ensure, that every other endpoint is able to update their formerly received device descriptions and new endpoints get the latest updates as well.

File: docs/communication.adoc [hunk 2]
```diff
@@ -49,6 +49,11 @@ The device descriptions are needed by the agrirouter to
 * filter for DDIs that are allowed to be sent to specific endpoints
 * be sent to endpoints, that subscribe for telemetry data, after the device description was already sent
 
+[IMPORTANT]
+====
+Please note that the device description has to be sent every time before the maschine starts sending live telemetry data to ensure, that each other endpoint is able to update their formerly received devices descriptions and new endpoints get the latest updates as well.
+====
```
Comment (Frank-Wiebeler) [inline docs/communication.adoc:55]: Please note that the device description has to be sent every time before the machine starts sending live telemetry data to ensure, that every other endpoint is able to update their formerly received device descriptions and new endpoints get the latest updates as well.

File: docs/messaging-workflow.adoc [hunk 3]
```diff
@@ -209,6 +209,11 @@ The device descriptions are needed by the agrirouter to
 * determine the relevant CU when sending a Message directly to a machine
 * filter for DDIs that are allowed to be sent to specific endpoints
 
+[IMPORTANT]
+====
+Please note that the device description has to be sent every time before the maschine starts sending live telemetry data to ensure, that each other endpoint is able to update their formerly received devices descriptions and new endpoints get the latest updates as well.
```
Comment (Frank-Wiebeler) [inline docs/messaging-workflow.adoc:214]: Please note that the device description has to be sent every time before the machine starts sending live telemetry data to ensure, that every other endpoint is able to update their formerly received device descriptions and new endpoints get the latest updates as well.

File: docs/tmt/efdi.adoc [hunk 4]
```diff
@@ -68,6 +68,10 @@ Make sure, that a timelog message never exceeds the limit of 1MB of data. If thi
 split this message into multiple timelog messages.
 ====
 
+[IMPORTANT]
+====
+Please note that the device description has to be sent every time before the maschine starts sending live telemetry data to ensure, that each other endpoint is able to update their formerly received devices descriptions and new endpoints get the latest updates as well.
```
Comment (Frank-Wiebeler) [inline docs/tmt/efdi.adoc:73]: Please note that the device description has to be sent every time before the machine starts sending live telemetry data to ensure, that every other endpoint is able to update their formerly received device descriptions and new endpoints get the latest updates as well.

File: docs/commands/endpoint.adoc [hunk 5]
```diff
@@ -230,11 +230,16 @@ Errors will be reported using `ACK_WITH_FAILURE`. For a list of possible errors,
 
 
 
-== Sending a teamset using device Descriptions
+== Sending a teamset using device descriptions
 
 === Definition
 
-The teamset describes a list of devices attached to the communication unit. The most common devices are agricultural machines connected to the CU as ISO11783-10 TaskControllers. For more information on ISO11783-10, please refer to https://aef-online.org
+The teamset describes a list of devices attached to the communication unit. The most common devices are agricultural machines connected to the CU as ISO11783-10 task controllers. For more information on ISO11783-10, please refer to https://aef-online.org
```
Comment (Frank-Wiebeler) [inline docs/commands/endpoint.adoc:237]: The teamset describes a list of devices attached to the communication unit. The most common devices are agricultural machines connected to the CU via an ISO11783-10 task controller client. For more information on ISO11783-10, please refer to https://aef-online.org

**Review by Frank-Wiebeler — CHANGES_REQUESTED** [review]
Good job so far, I just request some minor grammar changes

File: docs/commands/endpoint.adoc [hunk 6]
```diff
@@ -230,11 +230,16 @@ Errors will be reported using `ACK_WITH_FAILURE`. For a list of possible errors,
...
+== Sending a teamset using device descriptions
 
 === Definition
 
-The teamset describes a list of devices attached to the communication unit. The most common devices are agricultural machines connected to the CU as ISO11783-10 TaskControllers. For more information on ISO11783-10, please refer to https://aef-online.org
+The teamset describes a list of devices attached to the communication unit. The most common devices are agricultural machines connected to the CU via an ISO11783-10 task controller client. For more information on ISO11783-10, please refer to https://aef-online.org
+
+[IMPORTANT]
+====
+Please note that the device description has to send every time before the machine starts sending live telemetry data to ensure, that each other endpoint is able to update their formerly received devices descriptions and new endpoints get the latest updates as well.
```
Comment (Frank-Wiebeler) [inline docs/commands/endpoint.adoc:241]: ...has to be sent...

Comment (Frank-Wiebeler) [reply docs/commands/endpoint.adoc:241]: ...formerly received device descriptions ... (no s)

Comment (Frank-Wiebeler) [reply docs/commands/endpoint.adoc:241]: as mentioned, for new endpoints that should not be an issue, so we can remove "and new endpoints get the latest updates as well"

**Review by Frank-Wiebeler — CHANGES_REQUESTED** [review]
Some minor grammar topics; please adjust the text at the other added paragraphs as well

**Comment by oliverrahner** [comment]
Has been manually integrated into #113 in https://github.com/DKE-Data/agrirouter-interface-documentation/pull/113/commits/81888f9675bbdbf9e932c307bcf258352f98feaf

**Pull request closed without merging**